modules/snooker/src/test/ts/atomic/resize/RecalculationsTest.ts

```diff
@@ -37,25 +37,31 @@ UnitTest.test('RecalculationsTest', () => {
 
     Arr.each(expected, (expt) => {
       assert.eq(expt.columns, Arr.map(actualColumnWidth, (cell) => ({
-        element: cell.element,
+        element: TextContent.get(cell.element),
         width: cell.width
       })));
 
       assert.eq(expt.widths, Arr.map(actualCellWidth, (cell) => ({
-        element: cell.element,
+        element: TextContent.get(cell.element),
         width: cell.width
       })));
 
       assert.eq(expt.heights, Arr.map(actualCellHeight, (cell) => ({
-        element: cell.element,
+        element: TextContent.get(cell.element),
         height: cell.height
       })));
     });
   };
 
-  const makeDetail = (fakeEle: any, rowspan: number, colspan: number) => Structs.detail(fakeEle as SugarElement, rowspan, colspan);
-  const makeRow = (fakeEle: any, cells: Structs.Detail[]) => Structs.rowdata(fakeEle as SugarElement, cells, 'tbody');
-  const makeColumnGroup = (fakeEle: any, cols: Structs.Detail[]) => Structs.rowdata(fakeEle as SugarElement, cols, 'colgroup');
+  const createCell = (text: string): SugarElement => {
```

Comment:
```suggestion
  const createCell = (text: string): SugarElement<HTMLTableCellElement> => {
```

Reply:
Also, these tests should probably be moved to `browser` as they are no longer `atomic`